hack/generators/release-controllers/content/resources/rhel-7/golang/Dockerfile

```diff
@@ -4,11 +4,7 @@ ARG MAJOR
 ARG MINOR
 
 ENV VERSION=rhel-7-golang-openshift-${MAJOR}.${MINOR} \
-    GOCACHE=/go/.cache \
     GOARM=5 \
-    GOPATH=/go \
-    GOROOT=/usr/local/go \
-    GOFLAGS='-mod=vendor' \
```

Comment:
Removing this means that vendor isn't used by default anymore, I think that might surprise ppl?
/hold

Reply:
That flag is baked into the underlying ART builder image.

Reply:
Also, if ART changes GOFLAGS to something new for go 1.X, that should propagate to CI without modification.